examples/vg-specs/bar_grouped.vg.json

```diff
@@ -28,9 +28,7 @@
             "url": "data/population.json",
             "format": {
                 "type": "json",
-                "parse": {
-                    "people": "number"
-                }
+                "parse": {}
```

Comment:
Looks like it is https://vega.github.io/vega/docs/data/#format. But I can't tell for sure. 

Whatever the logic is, we do not want auto parse.